DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factory/AnonymizerSettingsFactory.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Exceptions;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Settings
+{
+    public class AnonymizerSettingsFactory : IAnonymizerSettingsFactory
+    {
+        public object CreateAnonymizerSetting(Type settingType, JObject settingObject)
```

Comment:
Are we using this method anywhere?

Reply:
Not used, just leave the api. Should we remove it?